src/data/store.js

```diff
@@ -24,6 +24,20 @@ function buildFindFirstLastQuery(queryBuilder, sortDir) {
   return DataQueryBuilder.toRequestBody(query)
 }
 
+function buildFindQuery(queryBuilder) {
```

Comment:
this logic is already persisted in the `DataQueryBuilder.toRequestBody` method

Reply:
\+